app/Community/Listeners/NotifyMessageThreadParticipants.php

```diff
@@ -90,8 +87,6 @@ private function forwardToDiscord(
         }
 
         $color = hexdec('0x0066CC');
-        $mentionRoles = collect(Arr::wrap($inboxConfig['mention_role'] ?? []))
-            ->map(fn ($role) => '<@&' . $role . '>');
```

Comment:
I've restored `$mentionRoles` in latest, but I've commented out the conditional logic responsible for attaching the roles to the message content.

We could certainly go the .env route, but I have a feeling that note pinging roles is likely going to be our long-term solution. Given that, it may make more sense to go code-first and have it documented in the code rather than relying on the lack of presence of specific env values.